source/fundamentals/enterprise-authentication.txt

```diff
@@ -297,6 +297,10 @@ see the corresponding syntax.
       The following code example shows how to specify Azure IMDS OIDC authentication. 
       Replace the ``<percent-encoded audience>`` placeholder with the percent-encoded
       value of the ``audience`` parameter configured on your MongoDB deployment. 
+
+      You can't pass values containing the comma (``,``) character to the ``authMechanismProperties`` 
```

Comment:
[note to reviewer] I noticed in our [style guide](https://www.mongodb.com/docs/meta/style-guide/writing/use-positive-statements/) we recommend using positive statements, but I thought this wording was clearer than saying something like "Avoid using...". However, I'm open to suggestions here!

Reply:
Agreed, I think this is more clear than saying something like "Avoid using..."